.travis.yml

```diff
@@ -18,41 +18,41 @@ matrix:
         - name: "linux-x86-clang-locks"
           os: linux
           compiler: clang
-          env: CONFIG_OPTS="--enable-swtok --enable-icsftok --enable-ccatok --enable-testcases --enable-locks --with-systemd" CFLAGS="-O3 -Wextra -std=c99 -pedantic -Werror -DDEBUG"
+          env: CONFIG_OPTS="--enable-swtok --enable-icsftok --enable-ccatok --enable-testcases --enable-locks --with-systemd" CFLAGS="-O3 -pedantic -Werror -DDEBUG"
```

Comment:
I would also move the `-pedantic` setting to configure.ac (here and in all other sections). Otherwise you're developing something and got findings in your work very late in the travis run. If you get warnings/errors earlier, you can fix it before creating the pull request.

Reply:
Yes maybe that's a good idea. 
But the `-Werror` is left out by intention, as one may want to compile while there are still warnings.